typings/globals.d.ts

```diff
@@ -1,34 +1,21 @@
-import {AsyncWrapBinding} from "./internalBinding/async_wrap";
-import {BlobBinding} from "./internalBinding/blob";
-import {ConfigBinding} from "./internalBinding/config";
-import {ConstantsBinding} from "./internalBinding/constants";
-import {HttpParserBinding} from "./internalBinding/http_parser";
-import {FsBinding} from "./internalBinding/fs";
-import {FsDirBinding} from "./internalBinding/fs_dir";
-import {MessagingBinding} from "./internalBinding/messaging";
-import {OptionsBinding} from "./internalBinding/options";
-import {OSBinding} from "./internalBinding/os";
-import {SerdesBinding} from "./internalBinding/serdes";
-import {SymbolsBinding} from "./internalBinding/symbols";
-import {TimersBinding} from "./internalBinding/timers";
-import {TypesBinding} from "./internalBinding/types";
-import {URLBinding} from "./internalBinding/url";
-import {UtilBinding} from "./internalBinding/util";
-import {WorkerBinding} from "./internalBinding/worker";
-import {ModulesBinding} from "./internalBinding/modules";
-
-declare type TypedArray =
-  | Uint8Array
-  | Uint8ClampedArray
-  | Uint16Array
-  | Uint32Array
-  | Int8Array
-  | Int16Array
-  | Int32Array
```

Comment:
Please reduce your change to only to the goal of this pull request. It's really hard to review it right now.

Reply:
@anonrig Thank you for your advice

I split it into https://github.com/nodejs/node/pull/54060/commits/b6f6827ff5ec3c20dc78b772170ba5f5d7365d70 and https://github.com/nodejs/node/pull/54060/commits/f606246b80bdf401906d9bc28b9d82efc0c49f52 commits. 

https://github.com/nodejs/node/pull/54060/commits/b6f6827ff5ec3c20dc78b772170ba5f5d7365d70 Main content of this PR, the Global type fix.

https://github.com/nodejs/node/pull/54060/commits/f606246b80bdf401906d9bc28b9d82efc0c49f52 Applied the lint.